boards/myboard/myfc/default.cmake

```diff
@@ -0,0 +1,105 @@
+px4_add_board(
+PLATFORM nuttx
+VENDOR myboard
+MODEL myfc
```

Comment:
We will need to nail these done at some point (The ripple can be large so sooner is better)

dir names comments and firmware.prototype and booloader QGC etc will have to change